src/app/tests/suites/certification/Test_TC_DISHALM_1_1.yaml

```diff
@@ -11,90 +11,109 @@
-
-          The list MAY contain values in the Manufacturer Extensible Identifier (MEI) range: (0xXXXX_0000 - 0xXXXX_4FFF), where XXXX is the allowed MEI range (0x0001 - 0xFFF1), these values SHALL be ignored.
-
-          The list SHALL NOT contain any values in the Test Vendor or invalid range: (0x0000_5000 - 0x0000_EFFF and 0x0000_FFFF), (0xXXXX_5000 - 0xXXXX_FFFF) and (0xFFF1_0000 - 0xFFFF_FFFF), where XXXX is the allowed MEI range (0x0001 - 0xFFF1)
-      disabled: true
-
-    - label: "Step 5: TH reads from the DUT the EventList attribute."
-      verification: |
-          Verify that the DUT response contains the EventList attribute and have the list of supported events:
-          The list SHALL include all the mandatory entries:
-          0x00
-          This cluster has no entries in the standard or scoped range.
-
-          The list SHALL NOT contain any additional values in the standard or scoped range: (0x0000_0000 - 0x0000_00FF)
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH reads the ClusterRevision attribute from the DUT"
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 1
+          constraints:
+              type: int16u
+
+    - label: "TH reads the FeatureMap attribute from the DUT"
+      PICS: " !DISHALM.S.F00 "
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 0
+          constraints:
+              type: bitmap32
+
+    - label: "TH reads the FeatureMap attribute from the DUT"
+      PICS: "DISHALM.S.F00"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x1]
+
+    - label: "TH reads the AttributeList attribute from the DUT"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [0, 2, 3, 65528, 65529, 65530, 65531, 65532, 65533]
+
+    - label: "TH reads the attribute(Latch) in AttributeList from the DUT"
+      PICS: DISHALM.S.F00
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [1]
+
+    - label: "Read the global attribute: EventList"
+      command: "readAttribute"
+      attribute: "EventList"
+      response:
+          value: [0]
+          constraints:
+              type: list
+
+    - label: "Read the global attribute: AcceptedCommandList"
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          value: []
```

Comment:
This does not match the test plan and will fail on any server that actually has commands it accepts, no?

Reply:
This is how other test yaml scripts with the same or similar verification handles the command so from my understanding it should not fail.

Reply:
Well, if you look at your CI failures:

```
ERROR   06:44:20.860 - TEST OUT  : CHIP:TOO:  ***** Test Step 7 : Read the global attribute: AcceptedCommandList
...
ERROR   06:44:20.868 - TEST OUT  : CHIP:TOO:  ***** Test Failure: acceptedCommandList value mismatch: expected only 0 items, but have more than that (actual value too long)
....
ERROR   Test_TC_DISHALM_1_1            - FAILED in 2.04 seconds
```

I don't know what other YAMLs you are looking at, but they are either different or testing clusters that do not in fact have commands....